### PR TITLE
7275 os account in ra

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java [hunk 1]
```diff
@@ -1116,6 +1142,34 @@ private boolean parseSamPluginOutput(String regFilePath, AbstractFile regAbstrac
             for (Map<String, String> userInfo : userSet) {
                 userInfoMap.put(userInfo.get(SID_KEY), userInfo);
             }
+            
+            // New OsAccount Code 
+            OsAccountManager accountMgr = tskCase.getOsAccountManager();
+            HostManager hostMrg = tskCase.getHostManager();
+            Host host = hostMrg.getHost(tskCase.getDataSource(regAbstractFile.getDataSourceObjectId()));
```
Comment (bcarrier) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java:1149]: We should be able to get the host from the DataSource that was passed into RecentActivity. It should have a getHost() method on it.

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java [hunk 2]
```diff
@@ -2138,4 +2192,277 @@ public void process(Content dataSource, IngestJobContext context, DataSourceInge
...
+     *
+     * @param file     Registry file
+     * @param sid      Account sid
+     * @param userName Login name
+     * @param homeDir  Account home Directory
+     *
+     * @throws TskCoreException
+     * @throws TskDataException
+     */
+    private void updateOsAccount(AbstractFile file, String sid, String userName, String homeDir) throws TskCoreException, TskDataException {
```
Comment (bcarrier) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java:2207]: I'd rename this createOrUpdateOsAccount to make it more clear.

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java (hunk 1 above)
Comment (kellykelly3) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java:1149]: I am sorry, I am not following.  Content doesn't seem to have a getHost method.

Comment (bcarrier) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java:1149]: Right, but DataSource does and it should be a DataSource object that is passed into RecentActivity.

**Review by APriestman — APPROVED** [review]

**Pull request merged**